model/http-common.yaml

```diff
@@ -63,6 +63,44 @@ groups:
           conditionally_required: If and only if one was received/sent.
         brief: '[HTTP response status code](https://tools.ietf.org/html/rfc7231#section-6).'
         examples: [200]
+      - id: error.reason
+        brief: 'Low-cardinality error reason.'
+        type:
+          allow_custom_values: true
+          members:
+            - id: empty
+              value: ""
+              brief: 'No error. Default value.'
+            - id: other
+              value: "_OTHER"
+              brief: 'Any error reason instrumentation does not define custom value for.'
+        requirement_level:
+          conditionally_required: If request or response has ended prematurely.
+        examples: ['timeout', 'name_resolution_error', 'server_certificate_invalid']
+        note: >
+          Instrumentations SHOULD define and document a list of applicable error reasons that describe why request
+          has failed or response was not received partially or fully.
+          For example, request could be canceled, DNS resolution or TLS handshake may fail,
+          or the connection maybe prematurely terminated before response is sent/received.
+
+          Error reason values MUST be a closed set known at the development time of the instrumentation
+          and MUST have low cardinality.
+
+          When missing, the value is assumed to be `empty`.
+
+          Request is considered to be completed prematurely if it was cancelled or an error occurred preventing
+          client or server from reading or writing request/response.
+
+          On the server side it means that at least of the following conditions has happened:
+          - request headers were not sent in full
```

Comment:
For some reason these didn't turn out as bullet points in the rendered markdown :(

Reply:
If you use ">" for the string, everything will be collapsed into a single line by the YAML parser. Use a pipe symbol "|" instead.